analyzers/src/SonarAnalyzer.Common/Rules/ParametersCorrectOrderBase.cs

```diff
@@ -56,9 +56,9 @@ public abstract class ParametersCorrectOrderBase<TSyntaxKind> : SonarDiagnosticA
                                 && Language.Syntax.ArgumentList(c.Node).FirstOrDefault(x => MatchingNames(parameterSymbol, ArgumentName(x))) is { })
                             {
                                 var secondaryLocations = methodParameterLookup.MethodSymbol.DeclaringSyntaxReferences
-                                    .Select(s => Language.Syntax.NodeIdentifier(s.GetSyntax())?.GetLocation())
+                                    .Select(s => Language.Syntax.NodeIdentifier(s.GetSyntax())?.GetLocation()?.ToSecondary())
```

Comment:
Does it make sense to have a `ToSecondary()` for `SyntaxToken` as well?

Reply:
We have one :)